#!/usr/bin/env python
PACKAGE = "sensor_fusion"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("leaf_size_x", double_t, 0, "X decimation", 0.1, 0.01, 1.0)
gen.add("leaf_size_y", double_t, 0, "Y decimation", 0.1, 0.01, 1.0)
gen.add("leaf_size_z", double_t, 0, "Z decimation", 0.1, 0.01, 1.0)

exit(gen.generate(PACKAGE, "sensor_fusion", "fusion"))